src/opera/util/mock_utils.py

```diff
@@ -1,22 +1,23 @@
 #!/usr/bin/env python3
 
 """
-============
-img_utils.py
-============
+=================
+mock_utils.py
+=================
 
-Image file utilities for use with OPERA PGEs.
+Utilities used to simulate the Geospatial Data Abstraction Library (GDAL).
 
 """
-import contextlib
-import io
-import os
-from collections import namedtuple
+
 from copy import deepcopy
-from datetime import datetime
-from functools import lru_cache
 from os.path import exists
 
+import mgrs
+
+
+S1_SLC_HDF5_PREFIX = ""
```

Comment:
This should be removed as only `h5_utils.py` needs it and it has its own version already.

Reply:
Done